website/templates/profile/account.mako

```diff
@@ -160,7 +160,8 @@
                                 >
                                 <p class="help-block" data-bind="validationMessage: passwordConfirmation" style="display: none;"></p>
                             </div>
-                            <button type="submit" class="btn btn-primary">Update password</button>
+                            ## TODO: change so that password strength submit validation happens in knockout on the form, not with this disable
```

Comment:
Could you make a jira ticket for this todo, then update the comment so that it's `## TODO: [OSF-XXXX] change…`?